.vscode/feat-test.code-snippets

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This makes it so in. vscode, you can use `ft` and it will create a snippet for a feature test. I was thinking of adding some auto-generation tooling but decided to create tests with this instead.

Reply:
Nice!